pkg/client/clientset/versioned/typed/operator.min.io/v1/fake/fake_minioinstance.go

```diff
@@ -1,18 +1,18 @@
-/*
-Copyright The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
+// This file is part of MinIO Operator
```

Comment:
As these files are still auto-generated, The copyrights need not be changed right?

Reply:
the `k8s.io/code-generator/hack/boilerplate.go.txt` allows you to specify the copyright for the auto-generated files